modules/gcp_bigquery_table/inputs.tf

```
@@ -28,3 +28,11 @@ variable "customer_managed_key_id" {
   type        = string
   description = "Describes the Cloud KMS encryption key that will be used to protect destination BigQuery table. The BigQuery Service Account associated with your project requires access to this encryption key."
 }
+
+variable "materialized_view" {
+  type = set(object({
+    query = string
+  }))
+  description = "create view for table"
+  default     = []
+}
```

Wondering: If this objects only contain one property (`query`), couldn't we simplify it and not use a set of objects, but a set of strings? Or maybe you plan to extend it in the future and add more properties?
```suggestion
variable "materialized_view_queries" {
  type = set(string)
  description = "Queries to materialize views"
  default     = []
}
```

---

There are more variables we can specify so having it as object can be added fields later